chapters/functions.tex

```diff
@@ -2398,6 +2398,11 @@ \subsection{Annotations for External Functions}\label{annotations-for-external-l
   {\lstinline!annotation(SourceDirectory="modelica:/ModelicaLibraryName/Resources/Source")!}\annotationindex{SourceDirectory}, gives the location for source files.
   The preceding one is the default and need not be specified; but another location could be specified by using an URI name for the source directory, see \cref{external-resources}.
   It is not specified how they are built.
+\item
+  The {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives the license files for the function.
+  The license files are human-readable files containing the license conditions.
```

Comment:
The annotation syntax only allows specifying a single file, which should be matched by the text:
```suggestion
  The {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives a license file for the function.
  License files are human-readable files containing the license conditions.
```

Reply:
This begs the question: Can a function have multiple `License`-annotations?

Reply:
I'll answer my own question by noting that it would probably be better to just refer to how the annotation's definition for a top-level class than to partially repeat the definition here:
```suggestion
  The {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives a license files for the function.
  It is analogous to the \lstinline!License!-annotation for a top-level class, see \cref{license-texts}.
```

Reply:
The last line is still useful.

Reply:
Seems resolved.